src/sink/wavefront.rs

```diff
@@ -335,6 +348,7 @@ mod test {
         assert!(lines.contains(&"test.timer.99 12.101 645181811 source=test-src"));
         assert!(lines.contains(&"test.timer.999 12.101 645181811 source=test-src"));
         assert!(lines.contains(&"test.timer.count 3 645181811 source=test-src"));
+        assert!(lines.contains(&"test.timer.mean 5.434333333333334 645181811 source=test-src"));
```

Comment:
It will fail to compile because of the constraints placed on the CKMS type `T`. It's a pain, but at least you can't sleep-walk into an accident.